pmd-java/src/main/java/net/sourceforge/pmd/lang/java/ast/ASTType.java

```diff
@@ -5,6 +5,15 @@
 
 package net.sourceforge.pmd.lang.java.ast;
 
+/**
+ * Represents a type reference.
+ * 
+ * <pre>
+ * Type ::= "var" | ReferenceType | PrimitiveType
```

Comment:
If you agree with the above comment, then this doc needs to be updated

Reply:
this should be updated then